Federico/chat improvements

federico-razzoli opened the pull request:
Add IRC link, write freenode lowercase for consistency with the original project.

Commit:
Make freenode lowercase consistently to how they write it

Commit:
Add channel name+link at the beginning

KarlLevik commented on chat/index.html:
```diff
@@ -9,7 +9,7 @@
   <h1>LAMP Database .ORG - Chat</h1>
 
-  <p>Is on freenode!</p>
+  <p>Is on freenode! Our channel is: <code><a href="irc://irc.freenode.net/lamp-database.org<">#lamp-database.org</a></code>.</p>
```
Is this less-than character a typo? 
...org<"...

federico-razzoli replied:
Yes. Fixed

KarlLevik reviewed (commented):
Looks good to me - except for that less-than character right after .org in the link! :)

shinguz commented:
I would say yes...
BR,
Oli

On 22/05/2020 23:54, Karl Levik wrote:
>
> *@KarlLevik* commented on this pull request.
>
> ------------------------------------------------------------------------
>
> In chat/index.html
> <https://github.com/lamp-database-org/lamp-database.org/pull/3#discussion_r429467836>:
>
> > @@ -9,7 +9,7 @@
>  
>    <h1>LAMP Database .ORG - Chat</h1>
>  
> -  <p>Is on freenode!</p>
> +  <p>Is on freenode! Our channel is: <code><a href="irc://irc.freenode.net/lamp-database.org<">#lamp-database.org</a></code>.</p>
>
> Is this a typo?
> ...org<"
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/lamp-database-org/lamp-database.org/pull/3#pullrequestreview-417203277>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ABZTKQ654HXU4IVJLQDVXU3RS3YB5ANCNFSM4NID3GZA>.
>

-- 

Go with the Flow: IG-Opencanoe Schweiz (http://ig-opencanoe.ch)
TopoDB - Sammlung aller Kletterrouten der Schweiz mit Topos (http://www.topodb.ch/).

Oli Sennhauser
Rebenweg 6
CH - 8610 Uster / Switzerland

Phone +41 79 830 09 33
e-Mail oli.sennhauser@bluewin.ch
Website http://www.shinguz.ch
Skype: oli_sennhauser

federico-razzoli commented:
Ooops. Fixed.

federico-razzoli commented:
Wait please, I'm having a weird problem with git.

Commit:
Delete line that appears different locally

Commit:
Re-enter lines that was seen different locally, to force desired change

federico-razzoli commented:
**Fixed, please check again.**

You may notice two strange commits. Explanation if you're curious:

- I made a commit to fix the typo, but I also introduced more stuff by misake
- I reverted it locally
- I've ran a push --force
- After that I saw the line with my fix locally, and the line without my fix remotely. Remote was correct, but the anomaly prevented me to make the change again
- So I deleted the line with a commit, and re-introduced the fixed line with the following commit.

KarlLevik reviewed (approved):
Looks good!
